WindowState/README.md

```diff
@@ -6,104 +6,213 @@ The component provides the current window information as a Compose state, includ
+
+## Add to your project
+
+1. Make sure you have **mavenCentral()** repository in your top level **build.gradle** file:
+
+    ```gradle
+    allprojects {
+        repositories {
+            google()
+            mavenCentral()
+         }
+    }
+    ```
+
+2. Add dependencies to the module-level **build.gradle** file (current version may be different from what's shown here).
+
+    ```gradle
+    implementation "com.microsoft.device.dualscreen:windowstate:1.0.0-alpha02"
+    ```
+
+3. Also ensure the compileSdkVersion and targetSdkVersion are set to API 31 or newer in the module-level build.gradle file.
+
+    ```gradle
+    android { 
+        compileSdkVersion 31
+        
+        defaultConfig { 
+            targetSdkVersion 31
+        } 
+        ... 
+    }
+    ```
+
+4. Access the info about the window state from **WindowState** to build or adjust your UI. Please refer to the [sample](https://github.com/microsoft/surface-duo-compose-sdk/tree/main/WindowState/sample) for more details.
+
+## API reference
 
 ```kotlin
 @Composable
 fun Activity.rememberWindowState(): WindowState
 ```
+
 An interface to provide all the relevant info about the device window.
 
+### Fold properties
+
+```kotlin
+val foldSizeDp: Dp
+```
+
+Returns a dp value of the thickness of the hinge of dual-screen device or the folding line of foldable device when the device is in dual-screen mode. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+```kotlin
+val foldablePane1SizeDp: Size
+```
+
+Returns the dp size of the primary pane of the dual-screen or foldable device when the device is in dual-screen mode. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+The primary pane is always the top pane and the left/right pane, depending on the local language layout direction.
+
+```kotlin
+val foldablePane2SizeDp: Size
+```
+
+Returns the dp size of the secondary pane of the dual-screen or foldable device when the device is in dual-screen mode. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+The secondary pane is always the bottom pane and the left/right pane, depending on the local language layout direction.
+
 ```kotlin
-val foldablePaneWidth: Int
+fun pane1SizeDp(pane1Weight: Float = 0.5f): Size
 ```
-Returns a pixel value of the width of a single pane of the dual-screen or foldable device when the device is in the dual-screen mode. If the device is in the single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+Returns the dp size of the primary pane of any device. This is the recommended value to use when creating dualscreen layouts, as it supports both large screens and foldables. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
```

Comment:
Can we highlight `any device, including the dual-screen, foldable device, and large-screen device?